doc/endpoints/track.md

```diff
@@ -8,30 +8,32 @@ This class provides methods to interact with the Spotify Web API and retrieve in
 
 This method is used to get information about a single track in the Spotify catalog, identified by its unique Spotify ID.
 
-**Endpoint**: [/tracks/{id}](https://developer.spotify.com/documentation/web-api/reference/get-track)
+#### Endpoint: [/tracks/{id}](https://developer.spotify.com/documentation/web-api/reference/get-track)
```

Comment:
Can you update the markdown so that when we click on the url, the webpage is opened in a new tab? Please do so for all the urls in this file

Reply:
it opens in a new tab for me